web_monitoring/internetarchive.py

```diff
@@ -34,6 +34,24 @@ class UnexpectedResponseFormat(WebMonitoringException):
 URL_CHUNK_PATTERN = re.compile('\<(.*)\>')
 DATETIME_CHUNK_PATTERN = re.compile(' datetime="(.*)",')
 
+def check_exists(lines):    
+    """
+    Check if Internet Archive has archived versions of a url.
+    """
+
+
+    try:
+        # The first three lines contain no information we need.
+        for _ in range(3):
+            next(lines)
+            
+
+    except StopIteration:
+        print("Internet archive does not have archived versions of this url.")
```

Comment:
I appreciate the clarification here of what is happening with `StopIteration`. To refine it a bit, I would make this sentence a code comment instead of a call to `print`. Library code should almost never print because it might "spam" the screen in the way that the application calling it does not want, and the application has no easy way of silencing the printing.